controllers/hibp.js

```diff
@@ -12,7 +12,16 @@ async function notify (req, res) {
   const subscribers = await DB.getSubscribersByHashes(hashes);
 
   const reqBreachName = req.body.breachName.toLowerCase();
-  const breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  let breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  if (!breach) {
+    // If breach isn't found, try to reload breaches from HIBP
+    await HIBP.loadBreachesIntoApp(req.app);
+    breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
```

Comment:
Not sure if we can find a better pattern for this, or hide it in a helper function somewhere...

I think this is now the 3rd or 4th time we're using similar code to filter breaches:

```sh
$ git grep -n locals.breaches.find

controllers/hibp.js:15:  const breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
controllers/scan.js:26:    featuredBreach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === req.body.featuredBreach.toLowerCase());
```